mithril-aggregator/src/database/provider/signer_registration.rs

```diff
@@ -67,7 +67,8 @@ impl From<SignerRegistrationRecord> for Signer {
     fn from(other: SignerRegistrationRecord) -> Self {
         Self {
             party_id: other.signer_id,
-            verification_key: other.verification_key,
+            verification_key: ProtocolSignerVerificationKey::from_json_hex(&other.verification_key)
```

Comment:
Use `try_into` instead of `ProtocolSignerVerificationKey::from_json_hex`. With this we don't need to import the type.